src/net/udp.rs

```diff
@@ -1,8 +1,10 @@
+use std::io;
+use std::net::{self, ToSocketAddrs, SocketAddr};
+use std::collections::HashMap;
+
 use bincode::{deserialize, serialize};
 use packet::Packet;
-use std::io;
-use std::net;
-use std::net::ToSocketAddrs;
+use super::*;
```

Comment:
Not a fan of using `*`, please import the specific modules etc

Reply:
Oke, I'll do that!